apps/mobile/src/components/PlanHero.tsx

```
@@ -0,0 +1,75 @@
+  const { result, interestSaved, monthsSaved } = plan;
+  const stuck = isPlanUnfinished(result);
+
+  return (
+    <Card className={isLoading ? 'opacity-70' : ''}>
+      <View className="mb-1 flex-row items-center justify-between">
+        <Eyebrow>Debt-free</Eyebrow>
+        {sample ? <Tag>Sample numbers</Tag> : null}
+      </View>
+      {stuck ? (
+        <Muted>
+          At {money(result.monthlyPayment)}/mo this plan doesn't pay off within 30 years. Raise the extra amount or a minimum to get a date.
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)